comment/templates/comment/comments/comment_form.html

```diff
@@ -8,8 +8,9 @@
 {% endif %}
 {% for field in comment_form.visible_fields %}
     {% if field.field.label == 'Content' %}
-    {% render_field field placeholder=placeholder class='js-comment-input form-control form-control-sm rounded-0' %}
+        {% render_field field placeholder=placeholder class='js-comment-input form-control form-control-sm rounded-0' %}
     {% else %}
-    {% render_field field class='js-comment-input form-control form-control-sm rounded-0' %}
+        {% render_field field class='js-comment-input form-control form-control-sm rounded-0' %}
     {% endif %}
 {% endfor %}
+<p class="error small text-danger mt-2 d-none">class</p>
```

Comment:
is `'class'` written here inside the `p` tag a mistake or do i miss something?

Reply:
😄 it was for styling the error. I will remove it.